### PR TITLE
[WIP] Make $PSScriptRoot description clearer

### PR DESCRIPTION
**Pull request by benmccallum**
As per [this SO answer](https://stackoverflow.com/a/17461595/725626), the current docs description is confusing.

Unless I'm missing something, as there could be cases where I'm wrong. But just throwing this suggestion out there.

Will update other versions' docs if/when the wording here is agreed upon.

# PR Summary
Tweaks wording to be clear that the value IS NOT the cwd, but the executing script's directory.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Sample scripts
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Language Spec
  - [ ] Legacy SDK

Uhh.... looks like "Reference" isn't an option.

**Cmdlet reference & about_ topics**
- [ ] Preview content
- [x] Version 7.1 content
- [ ] Version 7.0 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [ ] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [ ] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/7.1/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md [hunk 1]
```diff
@@ -484,7 +484,7 @@ objects in a pipeline.
 
 ### $PSScriptRoot
 
-Contains the directory from which a script is being run.
+Contains the full path of the executing script's directory.
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md:487]: just getting granular here but what do you think about:
```suggestion
Contains the full path of the executing script's parent directory.
```

I do think your version is much clearer than the original so I'm good with it.

**Comment by chasewilson** [comment]
Hey @benmccallum, have you had a chance to look at the suggested changes?

**Comment by benmccallum** [comment]
Sorry @chasewilson, have been away from GitHub notifications for a few days. Change is fine with me.

Do you want me to reflect this in other PS version docs?

**Comment by chasewilson** [comment]
Not a problem at all. I actually went ahead and made the change across the other versions. Thanks for submitting a PR. I like the new wording **much** better

**Pull request merged**